compiler/src/iree/compiler/Dialect/LinalgExt/IR/LinalgExtOps.td

```diff
@@ -539,45 +541,64 @@ def IREELinalgExt_AttentionOp : IREELinalgExt_Op<"attention",
     FlashAttention 2 and optionally results in the current `max` and `sum`
     statistics while processing the current tile.
 
-    If transpose_v is speciifed, the V tensor passed as input is assumed to
-    be transposed:
-
     attention(Q, K, V, scale) = softmax(Q @ K.T * scale) @ V.T
```

Comment:
Can you help update the description? This paragraph is not completed to me. E.g., new people would ask what `V.T` is when they look at the description.

Reply:
done! LMK if any of the notes need more details :)